core-bundle/contao/dca/tl_module.php

```diff
@@ -146,7 +146,7 @@
 			'options_callback'        => array('tl_module', 'getModules'),
 			'reference'               => &$GLOBALS['TL_LANG']['FMD'],
 			'eval'                    => array('helpwizard'=>true, 'chosen'=>true, 'submitOnChange'=>true, 'tl_class'=>'w50'),
-			'sql'                     => "varchar(64) COLLATE ascii_bin NOT NULL default 'navigation'"
+			'sql'                     => array('name'=>'type', 'type'=>'string', 'length'=>64, 'default'=>'navigation', 'customSchemaOptions'=>array('collation'=>'ascii_bin'))
```

Comment:
Why this change?

Reply:
Fixed in [0b8bb4f](https://github.com/contao/contao/pull/6232/commits/0b8bb4f887371998b6ab5d5efbe2a52d2b74d0b6)